source/stylesheets/vendors/_family.scss

```diff
@@ -187,3 +187,33 @@
   }
   @return $i
 }
+
+// Quantity Queries - http://quantityqueries.com/
+//////////////////
```

Comment:
I think I'll get rid of this. 
I'd like to keep the source file as light as possible. 

I'll mention the quantityqueries.com in the readme. How does it sounds for you ?